profiler/src/ProfilerEngine/Datadog.Profiler.Native/GenericSampler.h

```diff
@@ -0,0 +1,34 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+
+#pragma once
+
+#include "AdaptiveSampler.h"
+#include "IConfiguration.h"
+
+
```

Comment:
you can use `using namespace std::literals;` to use time unit like `2s` ...
and l.14, you can use `500ms` instead.

Reply:
✅ done